kerl

```diff
@@ -894,6 +899,17 @@ endif
 
 rehash
 ACTIVATE_CSH
+
+    otpver=$(get_otp_version "$rel")
```

Comment:
I agree. If it's ok to set this all the time, we should just set it all the time (if it's not already set.)